frigate/http.py

```diff
@@ -607,6 +608,44 @@ def stats():
     return jsonify(stats)
 
 
+@bp.route("/<camera_name>/ffprobe")
+def ffprobe(camera_name):
+    if camera_name not in current_app.frigate_config.cameras:
+        return jsonify(
+            {"success": False, "message": f"Camera name {camera_name} not found"}, "404"
+        )
+
+    config: CameraConfig = current_app.frigate_config.cameras[camera_name]
+
+    if len(config.ffmpeg.inputs) > 1:
```

Comment:
It seems like we would be better off always returning a list rather than conditionally altering the schema of the JSON that's returned from the endpoint.

Another approach could be to make this endpoint a more generic ffprobe endpoint and allow the frontend to pass the path. This would allow it to be used to test paths while using a UI config editor in the future.

Reply:
> Another approach could be to make this endpoint a more generic ffprobe endpoint and allow the frontend to pass the path. This would allow it to be used to test paths while using a UI config editor in the future.

Yeah I like that idea, will make it more accessible for general testing of camera feeds as well.